doc/howto/cluster_form.md

```diff
@@ -1,71 +1,264 @@
+
+```
+Would you like to use LXD clustering? (yes/no) [default=no]: yes
+What IP address or DNS name should be used to reach this server? [default=192.0.2.101]:
+Are you joining an existing cluster? (yes/no) [default=no]: no
+What member name should be used to identify this server in the cluster? [default=server1]:
+Setup password authentication on the cluster? (yes/no) [default=no]: no
+Do you want to configure a new local storage pool? (yes/no) [default=yes]:
+Name of the storage backend to use (btrfs, dir, lvm, zfs) [default=zfs]:
+Create a new ZFS pool? (yes/no) [default=yes]:
+Would you like to use an existing empty block device (e.g. a disk or partition)? (yes/no) [default=no]:
+Size in GiB of the new loop device (1GiB minimum) [default=9GiB]:
+Do you want to configure a new remote storage pool? (yes/no) [default=no]:
+Would you like to connect to a MAAS server? (yes/no) [default=no]:
+Would you like to configure LXD to use an existing bridge or host interface? (yes/no) [default=no]:
+Would you like to create a new Fan overlay network? (yes/no) [default=yes]:
+What subnet should be used as the Fan underlay? [default=auto]:
+Would you like stale cached images to be updated automatically? (yes/no) [default=yes]:
+Would you like a YAML "lxd init" preseed to be printed? (yes/no) [default=no]:
+```
+
+</details>
+
+After the initialization process finishes, your first cluster member should be up and available on your network.
+You can check this with `lxc cluster list`.
+
+### Join additional servers
+
+You can now join further servers to the cluster.
+
+```{note}
+The servers that you add should be newly installed LXD servers.
+If you are using existing servers, make sure to clear their contents before joining them, because any existing data on them will be lost.
+```
+
+To join a server to the cluster, run `lxd init` on the cluster.
+Joining an existing cluster requires root privileges, so make sure to run the command as root or with `sudo`.
+
+Basically, the initialization process consists of the following steps:
+
+1. Request to join an existing cluster.
+
+   Answer the first questions that `lxd init` asks accordingly:
+
+   - `Would you like to use LXD clustering?`
+
+     Select **yes**.
+   - `What IP address or DNS name should be used to reach this server?`
+
+     Make sure to use an IP or DNS address that other servers can reach.
+   - `Are you joining an existing cluster?`
+
+     Select **yes**.
+   - `Do you have a join token?`
+
+     Select **yes** if you configured the bootstrap server to use {ref}`authentication tokens <authentication-token>` (recommended) or **no** if you configured it to use a {ref}`trust password <authentication-trust-pw>`.
+1. Authenticate with the cluster.
+
+   There are two alternative methods, depending on which authentication method you choose when configuring the bootstrap server.
+
+   `````{tabs}
+
+   ````{group-tab} Authentication tokens
```

Comment:
Can we somehow add an emphasis here that the token method is the recommended method to use?